examples/CRISP/Dockerfile

```
@@ -48,17 +50,19 @@ RUN foundryup
 ENV PATH="/home/${USERNAME}/.risc0/bin:${PATH}"
 RUN curl -L https://risczero.com/install | bash
-RUN rzup install rust 1.81.0 
+RUN rzup install rust 1.85.0 
 RUN rzup install cpp 2024.1.5 
 RUN rzup install r0vm 2.0.0
 RUN rzup install cargo-risczero 2.0.0
 
+RUN sudo wget https://github.com/mikefarah/yq/releases/download/v4.45.1/yq_linux_amd64.tar.gz -O - |\
+  tar xz && sudo mv yq_linux_amd64 /usr/bin/yq
+
+RUN sudo mkdir -p /app && sudo chown -R $USERNAME:$USERNAME /app
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)